docker/scirius/bin/start-scirius.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
This is mostly some code from Stamus: https://github.com/StamusNetworks/scirius-docker/blob/master/django/scirius.sh so we can update copyright date.